### DISCUSSION TITLE
仓库和下载的zip不一致

### DISCUSSION THREAD

**Discussion by mudebug77** [community/community#109528] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

https://github.com/Nixdorfer/PalWorldSDK
这个仓库中，如果用 Download ZIP 按钮下载，得到的文件是错误的

错误的文件
![image](https://github.com/community/community/assets/34656102/e8abb3bc-9a2c-4c63-aed7-9f48cfc51054)

正确的文件
![image](https://github.com/community/community/assets/34656102/4ef2ab8e-2ab7-40cf-9744-266eeb89c33c)

**Comment by Akash1134** [comment]
Hi @mudebug77 , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
